rc-events-service/src/main/scala/com/epam/random_coffee/events/api/response/EventResponse.scala

```diff
@@ -0,0 +1,3 @@
+package com.epam.random_coffee.events.api.response
+
+case class EventResponse(id: Int, eventName: String)
```

Comment:
Add here companion object for mapping from common Event object like this:
object EventResponse {
  def fromEvent(event: Event): EventResponse = EventResponse(
   id = event.id,
   eventName = event,eventName
)
}

Reply:
companion is added